android/app/src/main/kotlin/com/jabook/app/jabook/compose/feature/player/BookmarksSheet.kt

```
@@ -0,0 +1,226 @@
+            }
+        }
+    }
+}
+
+private fun formatBookmarkDuration(ms: Long): String {
+    val totalSeconds = ms / 1000
+    val hours = totalSeconds / 3600
+    val minutes = (totalSeconds % 3600) / 60
+    val seconds = totalSeconds % 60
+    return if (hours > 0) {
+        String.format("%d:%02d:%02d", hours, minutes, seconds)
```

detekt (1.23.8)

[warning] String.format("%d:%02d:%02d", hours, minutes, seconds) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)